### ISSUE TITLE
screenscraper.fr - Login not work currently ?

### ISSUE THREAD

**Issue by DjDiabolik** [muldjord/skyscraper#359]
**Describe the bug**
After editing my current CONFIG.INI on screenscraper section i have uncommented the "userCreds=" value whit my current login.
i have put in this mode:
```
[screenscraper]
userCreds="myuser:mypassword"
```

Apparently when i lauch SkyScraper there's no login....... i can check that because whit an valid account the my daily quota limit it's 20.000 insted whitout login the same limit drop to 10.000.
I can see clearly the login it's not work...... infact i can see the limit it's set to 10.000.

**To Reproduce**
Add a login and try..... apparently there's no work.

**Technical information**
 - Skyscraper version: 3.7.7 compiled from source by retropie_setup.sh
 - Internet connection: Italian FTTH and my PC connected by LAN
 - OS: Ubuntu X64 installed on a Intel NUC VM40B
 - OS Version 22.04.1

**Comment by spilinek** [comment]
Screenscraper login is working for me. The daily limit is showing as 20,000. Also when I check Screenscraper site I see my scrapes for today is correct. 

I'm using Skyscraper 3.7.7-5 from this fork https://github.com/detain/skyscraper

I checked my config.ini and I'm also using:

```
[screenscraper]
userCreds="myuser:mypassword"
```

**Comment by spilinek** [comment]
I think you can add **--verbosity 3** to your Skyscraper command to get more details on why the login isn't working.

**Comment by DjDiabolik** [comment]
@spilinek so wait a moment but you have installed a potentially not ufficial fork of skyskraper ok ?

I look and it's apparently contain a most recent respect this source........ 

Anyway yes........ on my case i can see clearly the daily limit set as 10,000 and it's the cause i thinks my login it's not works.
For use --verbosity 3 i need to try to launch skyscraper not from retropie_setup but directly from terminal..... need some more test :)

**Comment by spilinek** [comment]
@DjDiabolik muldjord's README.md recommends the fork from detain:

> Pull Requests are not currently accepted as I am [focusing](https://github.com/muldjord/occurity) [on](https://github.com/muldjord/lithomaker) [other](https://github.com/muldjord/leddy) [projects](https://github.com/muldjord/boobox). If you are looking for a more community driven fork that seeks to expand the current feature set beyond what I've implemented here, you should check out [this fork](https://github.com/detain/skyscraper).

I was recently troubleshooting a request for screenscraper that wasn't working. The **--verbosity 3** parameter doesn't help much. I ended up modifying the code to print the URL request sent to screenscraper. Then I pasted the URL in a web browser to see the JSON response from screenscraper. Turns out there were no region names for the ROM. Workaround is to use **--region ss** where 'ss' is short for screenscraper. That will attempt to match the internal ROM name.

**Comment by DjDiabolik** [comment]
@spilinek the changed on readme.md it's slighty recent.
I have read this after i open this issue............. 

about using verbosity.. i don't understand it's can help me to understand because my correct login it's not accepted ?

why would it help to use of --region ss ?

**Comment by spilinek** [comment]
When you run Skyscraper do you see something like below? This should indicate that you've successfully logged into screenscraper.

> Fetching limits for user '**username**', just a sec...
> Setting threads to **6** as allowed for the supplied user credentials.

**Comment by DjDiabolik** [comment]
> When you run Skyscraper do you see something like below? This should indicate that you've successfully logged into screenscraper.
> 
> > Fetching limits for user '**username**', just a sec...
> > Setting threads to **6** as allowed for the supplied user credentials.

no... if i remember no....... i can see only the daily limit it's setted as 10.000 and this limit it's when you use the database whitout login user...

Whit login user the limit it's 20.000.....

Anyway i don't have a payed account so, in theory, i don't have the multithread enabled..... only the daily limit at 20.000 because i'm logged in :)
